db/migrate/20250619030503_set_cfp_data_upcoming_events.rb

```diff
@@ -0,0 +1,23 @@
+class SetCfpDataUpcomingEvents < ActiveRecord::Migration[8.0]
+  @@cfp_data = [
+    {slug: "picorubyoverflowkaigi-2025", cfp_close_date: "2025-07-15", cfp_link: "https://forms.gle/NsRZHW6MTV6aYPkF9"},
+    {slug: "rubyconf-taiwan-2025", cfp_close_date: "2025-11-05", cfp_link: "https://pretalx.coscup.org/coscup-2025/cfp"},
+    {slug: "friendly-rb-2025", cfp_close_date: "2025-07-01", cfp_link: "https://friendlyrb.com/cfp"},
+    {slug: "kaigi-on-rails-2025", cfp_close_date: "2025-06-30", cfp_link: "https://kaigionrails.org/2025/cfp/"},
+    {slug: "rocky-mountain-ruby-2025", cfp_close_date: "2025-06-30", cfp_link: "https://sessionize.com/rocky-mountain-ruby-2025/"},
+    {slug: "sfruby-2025", cfp_close_date: "2025-07-13", cfp_link: "https://cfp.sfruby.com/"},
+    {slug: "tiny-ruby-conf", cfp_close_date: "2025-07-30", cfp_link: "https://www.papercall.io/tinyruby"}
+  ]
```

Comment:
Is there a reason why you added a migration for this data instead of just putting it into the respective YAML files?